inst/examples/02-components.Rmd

```diff
@@ -596,6 +596,10 @@ Cross-references still work even when we refer to an item that is not on the cur
 
 ## Custom blocks
 
+:::{.rmdimportant latex=true}
+A new syntax is now adviced to create custom blocks. We invite you to refer to [Chapter 9.6 in the R Markdown Cookbook](https://bookdown.org/yihui/rmarkdown-cookbook/custom-blocks.html) [@rmarkdown2020] and no more use the `block` and `block2` engine described below. Those will be deprecated in future version of **bookdown**. Please make the change in your content if you are still using one of this engine directly.
```

Comment:
> A new syntax is now adviced to create custom blocks.

If an active voice can be used, the passive voice would not be used by me :)

If I can use an active voice, I would not use the passive voice.

Just kidding. This is no big deal, but I guess our editor may point it out before we print the 2nd edition of this book.